tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -192,6 +194,16 @@ func init() {
 	}
 	viper.SetDefault(constants.ProxyGID, "")
 
+	rootCmd.Flags().Bool(constants.RedirectDNS, false,
```

Comment:
why do we need this redirectDNS option?

Reply:
This isn't a change.  The logic for enabling this used to use an environment variable to enable/disable this (`ISTIO_META_DNS_CAPTURE`), but that doesn't work for CNI, so I added a command line option.  Viper should be initializing this option using the value from the env, if specified, when running in the init container.  This is basically synchronizing the existing iptables capabilities used by init container with cni.

Reply:
fwiw, this change is encapsulated in a separate commit.